tensorboard/webapp/metrics/store/metrics_types.ts

```diff
@@ -176,6 +177,8 @@ export interface MetricsNamespacedState {
   linkedTimeEnabled: boolean;
   stepSelectorEnabled: boolean;
   rangeSelectionEnabled: boolean;
+  singleSelectionHeaders: ColumnHeaders[];
+  rangeSelectionHeaders: ColumnHeaders[];
```

Comment:
It would be nice to namespace these so that different cards can have different sets of columns.

Because we are planning on putting the column controls on each card it seems confusing that changing one card would change ALL the cards.

Reply:
The current plan is to have the table columns always be the same for all cards. The main control will be in the settings panel. I see that it may be confusing for the dragging functionality that when you drag on one it changes in the other. But, that is the current design that Jesse and I came up with.

Reply:
Alright, I just wanted to make sure it was considered.